### PR TITLE
#9 Template strings

### PR DESCRIPTION
**Pull request by JakeRawr**

### CODE REVIEW

**Comment by taiga418** [comment]
I like this one!

File: README.md [hunk 1]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+```
+var firstName = 'Jake';
+var lastName = 'Rawr';
+console.log(`My name is ${firstName} ${lastName}`);
+// My name is Jake Rawr
+```
+
+You can do Multi-line strings without `\n` and simple logic (ie 2+3) inside `${}` in Template String.
+
+You are also able to to modify the output of template strings using a function; they are called Tagged template strings.
```
Comment (loverajoel) [inline README.md]: change by [Tagged template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings)

File: README.md [hunk 2]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+(see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings) for example usages of tagged template strings.
+
+You may also want to read https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/ to understand template strings more
```
Comment (loverajoel) [inline README.md]: [read](https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/)

File: README.md [hunk 3]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
 
+## #09 - Template Strings
+
+> 2016-01-09 by [@JakeRawr](https://github.com/JakeRawr)
+
+As of ES6, JS now has template strings as an alternative to the classic end quotes strings.
+
+Ex: 
+Normal string
+```
```
Comment (loverajoel) [inline README.md]: add ```javascript

File: README.md [hunk 4]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+var firstName = 'Jake';
+var lastName = 'Rawr';
+console.log('My name is ' + firstName + ' ' + lastName);
+// My name is Jake Rawr
+```
+Template String
+```
```
Comment (loverajoel) [inline README.md]: add ```javascript

**Comment by loverajoel** [comment]
Please at the final squash your commits. thanks

**Comment by JakeRawr** [comment]
I've made the changes. Thanks

**Comment by loverajoel** [comment]
ok, thanks for your contribution!

**Pull request merged**